integration/popular_docker_hub_images/popular_docker_images.bats

```diff
@@ -13,11 +13,11 @@ setup() {
 	clean_env
 }
 
-@test "[volume mkdir] create a directory in an aerospike container" {
-	tmpdir=$(mktemp -d)
-	docker run --rm --runtime=$RUNTIME -d -v $tmpdir:/opt/aerospike/data aerospike/aerospike-server timeout 5 /entrypoint.sh asd
-	[ -n ${tmpdir} ] || false
-	rm -rf $tmpdir
+@test "[insert data] insert data in an aerospike container" {
+	docker run --runtime=$RUNTIME -d --name aerospike aerospike/aerospike-server
+	sleep 10
```

Comment:
Is this waiting for aerospike to start up? How do we know this value is going to be reliable?

Reply:
this `sleep` is to wait for `aerospike-server` initialization

I guess something like this can replace the `sleep`

```sh
set +e
for i in $(seq 1 5); do
    docker run --rm --runtime=$RUNTIME -i aerospike/aerospike-tools aql -h $(docker inspect -f '{{.NetworkSettings.IPAddress}}' aerospike) -c "insert into test.foo (PK, foo) values ('123','any'); select * from test.foo"
    if [ $? == 0 ]; then
        set -e
        return 0
    fi
done
set -e
return 1
```